server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -447,9 +448,23 @@ private Server startServer(ProjectManager pm, CommandExecutor executor,
             @Override
             public HttpFile get(String path, Clock clock, @Nullable String contentEncoding) {
                 requireNonNull(path, "path");
-                return HttpFileBuilder.of(HttpData.of(StandardCharsets.UTF_8, server.defaultHostname()))
-                                      .setHeader(HttpHeaderNames.CONTENT_TYPE, MediaType.PLAIN_TEXT_UTF_8)
-                                      .build();
+                final Server s = server;
+                assert s != null;
+                final Builder<String, String> b = new Builder<>();
+                b.put("hostname", s.defaultHostname());
+                final String title = cfg.webAppTitle();
+                if (!isNullOrEmpty(title)) {
+                    b.put("title", title);
+                }
+                try {
+                    return HttpFileBuilder.of(HttpData.ofUtf8(Jackson.writeValueAsString(b.build())))
+                                          .setHeader(HttpHeaderNames.CONTENT_TYPE, MediaType.JSON_UTF_8)
+                                          .build();
+                } catch (JsonProcessingException e) {
+                    return HttpFileBuilder.of(HttpData.ofUtf8("{\"hostname\":\"" + s.defaultHostname() + "\"}"))
```

Comment:
I guess we will never reach here. How about just throwing an `Error`?

Reply:
Sounds good to me. :-)